app/v1/models/request/AddressModel.scala

```diff
@@ -44,52 +44,34 @@ object AddressModel {
   val startDatePath: JsPath = __ \ "startDate"
   val endDatePath: JsPath = __ \ "endDate"
 
-  private[models] def checkPostcodeMandated(postcode: Option[Postcode], countryCode: Option[String]): Option[Postcode] = {
-    countryCode.fold(postcode)(
-      countryCde => countryCde.toUpperCase match {
-        case "GBR" => postcode.fold({
-          Logger.warn("[AddressModel][checkPostcodeMandated] - postcode is required if country code is GBR")
-          throw new IllegalArgumentException("Postcode required if Country code is GBR")
-        })(postcode => Some(postcode))
-        case _ => postcode
-      }
-    )
+  private[models] def postcodeValidation(postcode: Option[Postcode], countryCode: Option[String]): Boolean = {
```

Comment:
This file needs to be reverted to what is in master as this code had leaked from another branch

Reply:
reverted to what is in the master